substrate/utils/frame/benchmarking-cli/src/storage/cmd.rs

```diff
@@ -116,6 +117,21 @@ pub struct StorageParams {
 	/// Include child trees in benchmark.
 	#[arg(long)]
 	pub include_child_trees: bool,
+
+	/// Enable PoV recorder.
+	///
+	/// The recorder has impact on performance when benchmarking with the TrieCache enabled.
+	/// It should be used for the final results for parachains.
+	#[arg(long)]
+	pub enable_pov_recorder: bool,
+
+	/// The batch size for the write benchmark.
+	///
+	/// Since the write size needs to also include the cost of computing the storage root, which is
+	/// done once at the end of the block, the batch size is used to simulate multiple writes in a
+	/// block.
+	#[arg(long, default_value_t = 1)]
+	pub batch_size: usize,
```

Comment:
I think we should use a sensible default like `100_000` here.

Reply:
Yes, will do.